### PR TITLE
Issue 15: Add Breaking News Header

### LINKED ISSUE #15: Add Breaking News Header
As a developer, I want to add a breaking news header to the Simulato test site in order to add more testable functionality

Acceptance Criteria:
- Header with breaking news stories added to site

(Referenced by QE-2087)

### PR DESCRIPTION
**Pull request by zmknox**
Closes #15.

Adds a breaking news header, which randomly chooses from available breaking news stories to display, or hides if there are none.

Also added a few extra default stories to show the random choosing of the header.

(Referenced by QE-2087)

### CODE REVIEW

File: src/breaking-news-header.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            }
+        }
+        
+        this.state = {
+            breakingNews: breaking,
+            item: breaking[Math.floor(Math.random() * breaking.length)],
+            show: false
+        };
+        if(breaking.length > 0) {
+            console.log("OKOK");
```
Comment (tomrdale) [inline src/breaking-news-header.js]: do you still need this console

File: src/breaking-news-header.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                item: breaking[Math.floor(Math.random() * breaking.length)]
+            });
+        }
+
+        this.openBreakingNews = this.openBreakingNews.bind(this);
+        this.handleClose = this.handleClose.bind(this);
+    }
+
+    openBreakingNews() {
+        console.log("ok");
```
Comment (tomrdale) [inline src/breaking-news-header.js]: ^

File: src/breaking-news-header.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    handleClose() {
+        this.setState({
+            item: this.state.breakingNews[Math.floor(Math.random() * this.state.breakingNews.length)],
+            show: false
+        });
+    }
+
+    render() {
+        if(this.state.breakingNews.length > 0) {
+            var x = this.state;
```
Comment (tomrdale) [inline src/breaking-news-header.js]: is this variable being used

**Review by tomrdale — CHANGES_REQUESTED** [review]

File: src/breaking-news-header.js (hunk 1 above)
Comment (zmknox) [reply src/breaking-news-header.js]: oops I forgot to remove that

**Review by tomrdale — APPROVED** [review]

**Pull request merged**